### PR TITLE
feat(owners): Display code owners for events

### PR DESCRIPTION
**Pull request by MaxBittker**
display owners for event
hacky :/ this component should be using our async component pattern and using the tool tip component, but in the interest of getting this ready to dogfood asap I opened this. I'll improve it tomorrow

### CODE REVIEW

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 1]
```diff
@@ -130,17 +157,58 @@ const SuggestedOwners = createReactClass({
...
   },
 
+  renderOwner(owner) {
+    let {rule} = this.state;
+    return (
+      <span
+        key={`${owner.id}:${owner.type}`}
+        className="avatar-grid-item tip"
+        onClick={() => this.assignToActor(owner)}
+        title={ReactDOMServer.renderToStaticMarkup(
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: 😬

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 2]
```diff
@@ -130,17 +157,58 @@ const SuggestedOwners = createReactClass({
...
+      >
+        <ActorAvatar actor={owner} />
+      </span>
+    );
+  },
+
   render() {
-    if (!(this.state.owners && this.state.owners.length)) {
+    let {committers, owners} = this.state;
+    let showOwners = new Set(this.getOrganization().features).has('internal-catchall');
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: If you're gonna use a mixin, there's `getFeatures`

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 3]
```diff
@@ -130,17 +157,58 @@ const SuggestedOwners = createReactClass({
...
     }
     return (
       <div className="m-b-1">
-        <h6>
-          <span>{t('Suggested Owners')}</span>
-          <small style={{background: '#FFFFFF'}}>Click to assign</small>
-        </h6>
-        <div className="avatar-grid">{this.state.owners.map(this.renderCommitter)}</div>
+        {committers.length ? (
+          <React.Fragment>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: Would be nice to refactor this into a method or component.

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx [hunk 4]
```diff
@@ -130,17 +157,58 @@ const SuggestedOwners = createReactClass({
...
+            <h6>
+              <span>{t('Suggested Owners')}</span>
+              <small style={{background: '#FFFFFF'}}>Click to assign</small>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: `t()`

**Review by billyvg — APPROVED** [review]
Would be nice to have some tests (I'm ok if we do a follow up PR for the other fixes).

File: src/sentry/static/sentry/app/components/group/suggestedOwners.jsx (hunk 1 above)
Comment (MaxBittker) [reply src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: is this that bad? better than building a string, right?

Comment (MaxBittker) [reply src/sentry/static/sentry/app/components/group/suggestedOwners.jsx]: it is gross that we send tooltips through bootstrap js

**Pull request merged**